hal/src/rtl872x/i2c_hal.cpp

```diff
@@ -210,8 +210,6 @@ class I2cClass {
             I2C_Cmd(i2cDev_, DISABLE);
             state_ = HAL_I2C_STATE_DISABLED;
             // RCC_PeriphClockCmd(APBPeriph_I2C0, APBPeriph_I2C0_CLOCK, DISABLE);
-            Pinmux_Config(hal_pin_to_rtl_pin(sdaPin_), PINMUX_FUNCTION_GPIO);
-            Pinmux_Config(hal_pin_to_rtl_pin(sclPin_), PINMUX_FUNCTION_GPIO);
             hal_gpio_set_drive_strength(sdaPin_, HAL_GPIO_DRIVE_DEFAULT);
```

Comment:
The code I added here  seems to be duplicated as we've set the drive strength in `hal_gpio_configure()`

Reply:
Should I remove lines 215 and 216 then?

Reply:
Yes, it is reconfigured, this is added by me in the overshoot PR, so it is fine